### PR TITLE
Admin Language Controls

### PR DESCRIPTION
**Pull request by Mnemotechnician**
# Description
Adds 5 new tricks that allow admins/testers to manipulate entities' languages and translators.

The tricks include:
- Add/remove universal language (UniversalLanguageSpeakerComponent, which overrides all default languages)
- Add/remove a specific language
- Edit a translator (spoken, understood, and required languages - all at once because QuickDialogSystem sucks)

This also adds a new supported field type to QuickDialogSystem - boolean, which is represented as a field that lets you input y/n, yes/no, true/false.

<details><summary><h1>Media</h1></summary>
<p>

https://github.com/user-attachments/assets/313a22c8-5191-464e-9638-ab1ce91ff34c

</p>
</details>

---

# Changelog
:cl:
- add: Added admin tricks to manipulate entities' languages and translators.

### CODE REVIEW

**Comment by Mnemotechnician** [comment]
I know there's an admin tooling CL but Iunno how to make the CL get added to it.

**Comment by DEATHB4DEFEAT** [comment]
> I know there's an admin tooling CL but Iunno how to make the CL get added to it.

The bot doesn't support admin CLs yet

File: Content.Shared/Administration/QuickDialogOpenEvent.cs [hunk 1]
```diff
@@ -132,4 +132,8 @@ public enum QuickDialogEntryType
     /// Maximum of 2,000 characters string.
     /// </summary>
     LongText,
+    /// <summary>
+    /// A string that is one of the following: yes, true, no, false.
```
Comment (DEATHB4DEFEAT) [inline Content.Shared/Administration/QuickDialogOpenEvent.cs:136]: Incorrect

**Review by DEATHB4DEFEAT — CHANGES_REQUESTED** [review]
Editing languages should be commands with autofill, the add/remove universal is fine as a trick.

**Comment by Mnemotechnician** [comment]
I can make them as commands, but I know from experience commands are extremely inconvenient to use.

**Pull request closed without merging**